source/includes/_chat_summary.md

```diff
@@ -529,6 +567,7 @@ https://api.localiqservices.com/client_reports/chat_summary/TEST_1?start_date=20
 |---|---|---|
 |campaigns|Object|Data for specified campaign. [Campaigns Object](#chatcampaigns)|
 |organic|Object|Data for organic. [Organic Object](#chatorganic)|
+|otherpaid|Object|Data for non organic. [Otherpaid Object](#chatotherpaid)|
```

Comment:
@martin-hristov-acc can I ask for a small change to these three lines?
campaigns: chat counts for chats attributed to paid campaigns
organic: chat counts for chats attributed to organic sources (Organic search, Direct Site Traffic, etc.)
otherpaid: chat counts for chats attributed to paid traffic by other media vendors